Doc/library/codecs.rst

```diff
@@ -1389,7 +1389,7 @@ mapping.  It is not supported by :meth:`str.encode` (which only produces
 +--------------------+---------+---------------------------+
 | Codec              | Aliases | Purpose                   |
 +====================+=========+===========================+
-| rot_13             | rot13   | Returns the Caesar-cypher |
+| rot_13             | rot13   | Returns the Caesar-cipher |
```

Comment:
Let's leave this unchanged since `cypher` is a valid word.

Reply:
Thanks, I took reference from https://en.m.wikipedia.org/wiki/Caesar_cipher and I think I couldn't find Cypher across the docs making the change. I will revert it.

Reply:
@willingc kindly confirm if the change  needs to be reverted given the above link.

Thanks

Reply:
@tirkarthi Both are used interchangeably in the literature. I would leave the original author's spelling. Thanks.

Reply:
@willingc Sure. Reverted to cypher. Thanks.